python/ray/dag/tests/experimental/test_execution_schedule_gpu.py

```diff
@@ -145,37 +144,8 @@ def test_simulate_pp_2workers_2batches_1f1b(
         dag = MultiOutputNode([batch_1, batch_2])
     compiled_dag = dag.experimental_compile()
 
-    w1_expected_schedule = [
-        (0, _DAGNodeOperationType.READ),
-        (0, _DAGNodeOperationType.COMPUTE),
-        (0, _DAGNodeOperationType.WRITE),
-        (1, _DAGNodeOperationType.READ),
-        (1, _DAGNodeOperationType.COMPUTE),
-        (1, _DAGNodeOperationType.WRITE),
-        (2, _DAGNodeOperationType.READ),
-        (2, _DAGNodeOperationType.COMPUTE),
-        (3, _DAGNodeOperationType.READ),
-        (2, _DAGNodeOperationType.WRITE),
-        (3, _DAGNodeOperationType.COMPUTE),
-        (3, _DAGNodeOperationType.WRITE),
-        (4, _DAGNodeOperationType.READ),
-        (4, _DAGNodeOperationType.COMPUTE),
-        (4, _DAGNodeOperationType.WRITE),
-    ]
-    w2_expected_schedule = [
-        (0, _DAGNodeOperationType.READ),
-        (0, _DAGNodeOperationType.COMPUTE),
-        (0, _DAGNodeOperationType.WRITE),
-        (1, _DAGNodeOperationType.READ),
-        (1, _DAGNodeOperationType.COMPUTE),
-        (1, _DAGNodeOperationType.WRITE),
-        (2, _DAGNodeOperationType.READ),
-        (2, _DAGNodeOperationType.COMPUTE),
-        (2, _DAGNodeOperationType.WRITE),
-        (3, _DAGNodeOperationType.READ),
-        (3, _DAGNodeOperationType.COMPUTE),
-        (3, _DAGNodeOperationType.WRITE),
-    ]
+    w1_expected_schedule = [0, 1, 2, 5, 3, 4, 7, 6, 8]
```

Comment:
Please add a comment explaining what the expected schedule is.

Also, I assume there was no behavior change in this test?